product_docs/docs/edbcloud/beta/getting_started/02_connect_cloud_account.mdx

```diff
@@ -10,13 +10,15 @@ This topic describes how to set up your EDB Cloud account on Azure Marketplace.
 
 1. Within Azure Active Directory, ensure your role is owner and your user type is member (not guest) for the subscription you are using.
 
-1. Create an Azure Active Directory Application client to delegate Identity and Access Management functions to Azure Active Directory (AD).
+1. Create an Azure Active Directory Application client to delegate Identity and Access Management functions to Azure Active Directory (AD). You can create the Azure Active Directory Application using the Azure Portal. Users of the Azure CLI can create the Azure Active Directory Application using the `create-spn` script (see [Create Azure Active Directory Application Using `create-spn`](#create-azure-active-directory-application-using-create-spn)).
```

Comment:
We should probably recommend the use of `create-spn` somewhat more strongly here - it's going to be less error-prone.